src/test/java/org/broadinstitute/hellbender/tools/walkers/contamination/CalculateContaminationIntegrationTest.java

```diff
@@ -0,0 +1,55 @@
+package org.broadinstitute.hellbender.tools.walkers.contamination;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by David Benjamin on 2/16/17.
+ */
+public class CalculateContaminationIntegrationTest extends CommandLineProgramTest {
+    @Test
+    public void test() {
+        final String contig = "chr1";
+        final int spacing = 100000;
+        final double contamination = 0.05;
+        final double alleleFrequency = 0.2;
+        final int depth = 100;
+
+        final List<PileupSummary> ps = new ArrayList<>();
+
+        // if allele frequency is 0.2, then hets are about 10 times as common as hom var
+        for (int n = 0; n < 1000; n++) {
+            int position = n * spacing;
+                ps.add(n % 10 == 0
+                        ? new PileupSummary(contig, position, (int) (depth*contamination), (int) (depth * (1 - contamination)), 0, alleleFrequency)
+                        : new PileupSummary(contig, position, depth / 2, depth/2, 1, alleleFrequency));
+        }
+
+        // same idea, but loss of heterozygosity -- these sites should be thrown out
+        for (int n = 1000; n < 1100; n++) {
+            int position = n * spacing;
+            ps.add(n % 10 == 0
+                    ? new PileupSummary(contig, position, (int) (depth*contamination), (int) (depth * (1 - contamination)), 0, alleleFrequency)
```

Comment:
...or you didn't. Most of variants for n in (1000,1100) are thrown out as loss of heterozygosity as expected. But how? How can having one LOH in every 10 variants drastically reduce the `hetRatio` for _all_ variants nearby such that they are _all_ considered_ LOH?